scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,223 @@
+import sys
+
+import python_utils
+
+from . import fully_covered_files
+
+LCOV_FILE_PATH = os.path.join(os.pardir, 'karma_coverage_reports', 'lcov.info')
+PR_LCOV_FILE_PATH = os.path.join(os.curdir, 'tmp', 'pr-lcov.info')
+FULLY_COVERED_TESTS = fully_covered_files.fully_covered_files_array
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess."""
+    subprocess.check_call([
+        'python', '-m', 'scripts.run_frontend_tests'])
+
+
+def create_tmp_folder():
+    """Creates a temporary folder."""
+    if not os.path.exists('./tmp'):
+        os.mkdir('./tmp')
+
+
+def delete_tmp_folder():
+    """Delete the temporary folder."""
+    if os.path.exists('./tmp'):
+        shutil.rmtree('./tmp')
+
+
+def filter_lines(line):
+    """Check if the line has file path (SF) or total lines (LF) or covered
+    lines (LH) of the test.
+
+    Args:
+        line: str. A line from lcov file.
+
+    Returns:
+        Boolean. If the line has the file path or total lines or covered lines
+        of the test.
+    """
+    return ('SF' in line or 'LH' in line or 'LF' in line)
+
+
+def get_test_file_name(test_path):
+    """Get the file name from the absolute path.
+
+    Args:
+        test_path: string. The file's absolute path.
+
+    Returns:
+        String. The file path.
+    """
+    if not test_path:
+        sys.stderr.write(
+            'The test path is empty or null.'
+            'It\'s not possible to diff the test coverage correctly.')
+        sys.exit(1)
+
+    file_name = os.path.split(test_path)[1]
+    return file_name
+
+
+def get_lcov_file_tests(file_path):
+    """Get all tests from a lcov file, and filters it to return only:
+    - File path
+    - File total lines
+    - File covered lines
+
+    Args:
+        file_path: string. The path of lcov file.
+
+    Returns:
+        String array. An array with all tests filtered, including only important
+        data for the diff.
+    """
+    with python_utils.open_file(file_path, 'r') as f:
+        tests_array = f.read().split('end_of_record')
+        tests_array_filtered = []
+
+        for test in tests_array:
+            lines = [line for line in test.splitlines() if filter_lines(line)]
+            if len(lines) > 0:
+                tests_array_filtered.append(lines)
+
+        return tests_array_filtered
+
+
+def get_coverage_dict_tests():
+    """Build a dict with all covered files from develop branch.
+
+    Returns:
+        Dictionary. A dict containing file path, total lines and covered lines
+        of each tested file.
+
+    Raises:
+        Exception: If DEV_LCOV_FILE_PATH doesn't exist.
+    """
+    coverage_dict = {}
+
+    if not os.path.exists(PR_LCOV_FILE_PATH):
+        raise Exception(
+            'File at path {} doesn\'t exist'.format(PR_LCOV_FILE_PATH))
+
+    tests = get_lcov_file_tests(PR_LCOV_FILE_PATH)
+    for lines in tests:
+        total_lines = re.match('(\d+)', lines[1].split(':')[1]).group(1)
```

Comment:
Do not reference arbitrary indexes in `lines` -- this seems to indicate that `lines` should instead be a domain object.

Reply:
Sorry I didn't get it.

Reply:
This too, we discussed in the meeting. She is going to do it.

Reply:
Done.